### PR TITLE
feature: S3C-1968: kmip highlevel driver

### PR DESCRIPTION
**Pull request by ploki**

### CODE REVIEW

**Comment by ploki** [comment]
Not to be reviewed yet. 3 PRs have to be merged before paying attention to this one (You can take a look at #641 )

**Comment by ploki** [comment]
@bert-e reset

**Comment by ploki** [comment]
@bert-e reset

File: tests/utils/kmip/LoopbackServerChannel.js [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+        value: 'Gangsta Grade',
+    },
+    {
+        name: 'Prefered Algorithm',
+        tag: 0x542008,
+        type: 7,
+        value: 'Kevin Bacon',
+    },
+    {
+        name: 'Yo Dawg',
```
Comment (philipyoo) [inline tests/utils/kmip/LoopbackServerChannel.js:37]: yo

**Review by philipyoo — APPROVED** [review]

**Review by bennettbuchanan — APPROVED** [review]

**Comment by ploki** [comment]
@bert-e approve

**Pull request merged**